notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -3186,6 +3186,63 @@ the scientist profession [Dr. Herbert West and Dr. Howard Phillips were experime
 disembodied head [Laura's disembodied head was taunting Howard toward the film's conclusion.]
 
 
+movie: Save the Green Planet (2003)
+===================================
+
+:: Title
+Save the Green Planet!
+
+:: Description
+Save the Green Planet! (Korean title: 지구를 지켜라!, Jigureul Jikyeora!) is a
+South Korean science fiction comedy film written and directed by Jang
+Joon-hwan, released on 4 April 2003.
+
+Synopsis: A deranged man kidnaps another man, convinced that the latter is a
+space alien.
+
+:: Date
+2003-04-04
+
+:: References
+https://en.wikipedia.org/wiki/Save_the_Green_Planet!
+
+:: Ratings
+5 <paul>
+
+:: Major Themes
+what if the fate of the world was in my hands [The film's main character Byeong-gu believed, as it turned out correctly, that aliens from Andromeda were about to attack Earth and that he was the only one who could prevent them.]
```

Comment:
"as it turns out correctly" becomes ambiguous - not sure how it is best written but perhaps "... (as it turns out, correctly) ..." would be the safest course?

Reply:
Good eye. Reformulated according to your suggestion.